Add config flow for Times of day (#22143)
home-assistant/home-assistant.github.io: source/_integrations/tod.markdown

ha_release: 0.89
ha_iot_class: Local Push
ha_quality_scale: internal
ha_config_flow: true
ha_domain: tod
ha_platforms:
  - binary_sensor
---

> The Times of the Day integration provides a binary sensor which get their values by checking if the current time is within defined time ranges.

Reviewer: Choose one of:
- *provides a binary sensor which gets its value*
- *provides binary sensors which get their values*

Author: Almost... `which` -> `that` :) 

Thanks for spotting that one 👍 
```suggestion
The Times of the Day integration provides a binary sensor that gets its values by checking if the current time is within defined time ranges.
```

{% include integrations/config_flow.md %}
{% configuration_basic %}
Name:

>   description: The name the binary sensor should have. You can change it again later.

Reviewer: ```suggestion
  description: The name the binary sensor should have. This can be changed later.
```

Should we also mention that on and off-times can be changed?
